### PR TITLE
Added naming guidelines.

### PR DESCRIPTION
**Pull request by hannanelson**
Please check the server naming guidelines.

### CODE REVIEW

**Review by raogautham — DISMISSED** [review]
Good. Thanks!

**Review by aljrubior — APPROVED** [review]
LGTM

**Review by dhirsch1 — DISMISSED** [review]

File: modules/ROOT/pages/servers-create.adoc [hunk 1]
```diff
@@ -38,7 +38,10 @@ The registration token includes your organization's ID and the current environme
 . Paste the command on the command line.
 . Change the instance name `server-name` to the unique name to use to label Mule in the Runtime Manager console.
 +
-The `server-name` value is unique across environments. If the same organization exists in different environments, the `server-name` can be the same in those organizations.
+Server names can contain up to 60 alphanumeric characters (a-z, A-Z, 0-9), periods (.), and hyphens (-), but not spaces or other characters.
```
Comment (aljrubior) [inline modules/ROOT/pages/servers-create.adoc]: Can we please add the Underscore character `_` ?

Currently in prod we found the following non-alphanumeric characters in Server's names (Server count):
* `_` 3722
* `.` 4722
* `-` 53010

IMPORTANT!
We allow the use of Japanese characters in Server's and Application's name.

Ex. The following names are valid in our platform
* Sever: 日本のサーバー
* Application: 日本のアプリケーション

**Review by aljrubior — CHANGES_REQUESTED** [review]

**Review by aljrubior — APPROVED** [review]
LGTM

**Review by aljrubior — APPROVED** [review]
LGTM

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**